R/geom_phylopic.R

```diff
@@ -14,11 +14,11 @@ phylopic_env <- new.env()
 #'   specifies the height of the silhouettes in the units of the y axis. The
 #'   aspect ratio of the silhouettes will always be maintained.
 #'
-#'   The `color` (default: "black"), `fill` (default: NA), and `alpha` (default:
-#'   1) aesthetics can be used to change the outline color, fill color, and
-#'   transparency (outline and fill) of the silhouettes, respectively. If
-#'   `color` is specified and `fill` is NA the outline and fill color will be
-#'   the same. If "original" is specified for the `color` aesthetic, the
+#'   The `color` (default: NA), `fill` (default: "black"), and `alpha` (
+#'   default: 1) aesthetics can be used to change the outline color, fill color,
+#'   and transparency (outline and fill) of the silhouettes, respectively. If
+#'   `color` is specified and `fill` is NA `color` will be used as the fill
```

Comment:
Again, I am not sure this is behaving as expected:
```
library(ggplot2)
df <- data.frame(x = c(2, 4), y = c(10, 20),
                 name = c("Felis silvestris catus", "Odobenus rosmarus"))
ggplot(df) +
  geom_phylopic(aes(x = x, y = y, name = name), fill = NA, color = "blue", size = 10) +
  facet_wrap(~name) +
  coord_cartesian(xlim = c(1,6), ylim = c(5, 30))
```
![image](https://github.com/palaeoverse-community/rphylopic/assets/41071747/86484e80-95bd-4dad-895f-7de06c452753)